index.html

```diff
@@ -0,0 +1,205 @@
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/github.png" alt="github icon"></a></div></li>
+                    <li><div class="social-icon"><a href="#"><img src="images/Icons/angellist.png" alt="angellist icon"></a></div></li>
+                </ul>
+                
+               <div class="action"><a href="#">
+               <img src="images/Icons/Disabled.png" alt="Disabled"></a></div>
+        </section>
+
+            <div class="Primary"><h2 class="Primary-1">My Recent Works</h2><hr></div>
+        <section id="section-2" class="grid">
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
+                    <div class="btn"><a href="#" class="Primary-btn"> See Project </a></div>  
+              </div>
+              
+            </div>
+            <div class="card">
+                <div class="textholder">
+                    <h3>Multi-Post Stories Gain+Glory</h3>
+                    <div class="cells">
+                        
+                        <a class="cell" href="#"> Ruby on the rails</a>
+                        <a class="cell" href="#"> Javascript</a>
+                        <a class="cell" href="#"> Css</a>
+                        <a class="cell" href="#"> Html</a>
+                   </div>
```

Comment:
Same thing here
- [ ] On of the requirements says that the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `<ul>` and each of them a `<li>`. Just like this :point_down: 

```
 <ul class="cells">
      <li> <a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a>></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
      <li><a class="cell2" href="#"> CodeKit</a></li>
</ul>
```